src/components/utils/OffchainSigner/ExternalStorage.ts

```diff
@@ -1,21 +1,55 @@
+import { toSubsocialAddress } from '@subsocial/utils'
 import { createStorageKey } from 'src/utils/storage'
 import store from 'store'
 
-const OFFCHAIN_ADDRESS_KEY = 'df.OffchainAddress'
-const PROXY_ADDRESS_KEY = 'df.ProxyAddress'
-const OFFCHAIN_TOKEN_KEY = 'df.OffchainToken'
-const isCurrentOffchainAddress = (myAddress: string) =>
-  store.get(createStorageKey(OFFCHAIN_ADDRESS_KEY, myAddress)) === 1 ? true : false
+const SIGNER_ADDRESS_KEY = 'SignerAddress'
+const PROXY_ADDRESS_KEY = 'ProxyAddress'
+const SIGNER_TOKEN_KEY = 'SignerToken'
+const SIGNER_REFRESH_TOKEN_KEY = 'SignerRefreshToken'
+const TEMP_REGISTER_ACCOUNT = 'TempRegisterAccount'
+const SECRET_KEY = 'SecretKey'
```

Comment:
Add prefix for keys, like `Signer.SecretKey`,

Reply:
So it will be `df.Signer.SecretKey`, is that okay? @olehmell